tests/integration/run.sh

```diff
@@ -17,12 +19,14 @@ echo ''
 echo -e "\033[0;36m#\033[0m"
 echo -e "\033[0;36m# Starting PHP webserver\033[0m"
 echo -e "\033[0;36m#\033[0m"
-PHP_CLI_SERVER_WORKERS=3 php -S localhost:8080 -t ${ROOT_DIR} &
+touch phpserver.log
+PHP_CLI_SERVER_WORKERS=3 php -S localhost:8080 -t ${ROOT_DIR} &> phpserver.log &
 PHPPID1=$!
 echo -e "Running on process ID: \033[1;35m$PHPPID1\033[0m"
 
-# also kill php process in case of ctrl+c
-trap 'pkill -P $PHPPID1; kill -TERM $PHPPID1; wait $PHPPID1' TERM
+# Output filtered php server logs
+tail -f phpserver.log | grep --line-buffered -v " Accepted" | grep --line-buffered -v " Closing" &
```

Comment:
```suggestion
tail -f phpserver.log | grep --line-buffered -v -E ":\d+ Accepted$" | grep --line-buffered -v -E ":\d+ Closing$" &
```

Not sure regex works in `grep -E` the same way, but filtering out based on the word only looks a bit risky, so let's try to make sure it is really only the word and before there is a port?

Edit: Or it's okay, because all other output is actually from behat and not the php build in server.

Can you do the same on the federation build in server below (line 31/35)

Reply:
Can you do the same on the federation build in server below (line 31/35)
Maybe to a separate file for easier debugging

Reply:
> Edit: Or it's okay, because all other output is actually from behat and not the php build in server.

Yes, it's only from the PHP server, but we can match with the suggested regex to be sure!

> Can you do the same on the federation build in server below (line 31/35)

Yep, good idea.

Reply:
Not completely sure why `\d` isn't working on the runners, but `[0-9]` is working fine, so using this.